### PR TITLE
update readme and appinfo to feature .ai files

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

attempt to fix https://github.com/nextcloud/files_pdfviewer/pull/196#issuecomment-673677969

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -5,10 +5,10 @@
...
 	<description><![CDATA[This application integrates the PDF.js library into Nextcloud. Using this
-application users can view their PDF files online without the need to
+application, users can view their PDF documents and Illustrator® (.ai) files online without the need to
 download the file.
 
-When this application is enabled publicly shared PDF documents will also
+When this application is enabled, publicly shared PDF documents and Illustrator® (.ai) files will also
 get shown in the PDF viewer instead of only showing a single static
 snapshot of the document. The PDF viewer requires a modern browser and
 will not work with Microsoft® Internet Explorer® versions below 9.
```
Comment (skjnldsv) [inline appinfo/info.xml:14]: @jancborchardt this whole description seems a bit too technical, no?
Wanna change it?

**Comment by skjnldsv** [comment]
@szaimen thanks! Remember to put labels ;)

File: appinfo/info.xml (hunk 1 above)
Comment (jancborchardt) [reply appinfo/info.xml:14]: How about:
Summary: "PDF viewer"
Description: Directly view PDF documents in your Nextcloud. As an added bonus, this app can also display Adobe Illustrator  (.ai) files. Powered by pdf.js

Comment (skjnldsv) [reply appinfo/info.xml:14]: Looks good to me? Can you edit it? :wink:

Comment (skjnldsv) [reply appinfo/info.xml:14]: @szaimen :)

Comment (szaimen) [reply appinfo/info.xml]: Done 👍

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**